pysnmp/crypto/aes.py

```diff
@@ -0,0 +1,67 @@
+"""
+Crypto logic for RFC3826.
+
+https://tools.ietf.org/html/rfc3826
+"""
+from pysnmp.crypto import backend, CRYPTODOME, CRYPTOGRAPHY, generic_decrypt, generic_encrypt
+
+if backend == CRYPTOGRAPHY:
+    from cryptography.hazmat.backends import default_backend
+    from cryptography.hazmat.primitives.ciphers import algorithms, Cipher, modes
+elif backend == CRYPTODOME:
+    from Cryptodome.Cipher import AES
+
+
+def _cryptodome_cipher(key, iv):
+    """Build a Pycryptodome AES Cipher object.
+
+    :param bytes key: Encryption key
+    :param bytes IV: Initialization vector
+    :returns: DES3 Cipher instance
```

Comment:
This is AES, not DES

Reply:
Whoops; I must have copy-pasted those docstrings...